### PR TITLE
Azure Release 17.0.0

### PR DESCRIPTION
**Pull request by T-Kukawka**

### CODE REVIEW

**Comment by T-Kukawka** [comment]
e2e and manual tests green

File: azure/v17.0.0/release.yaml [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+    version: 3.12.0
+  - name: kubernetes
+    version: 1.22.6
+  - name: containerlinux
+    version: 3033.2.2
+  - name: calico
+    version: 3.21.3
+  - name: etcd
+    version: 3.4.18
+  date: "2022-03-14T14:43:09Z"
```
Comment (fiunchinho) [inline azure/v17.0.0/release.yaml:61]: I'm not sure if we update this field right before releasing.

**Review by fiunchinho — APPROVED** [review]

**Review by whites11 — APPROVED** [review]

**Pull request merged**